wallet/README

```diff
@@ -85,6 +85,7 @@ To upload the app to your Android device, use
 
 	# first time only
 	sudo apt-get install android-tools-adb
+	Make sure "Developer options" and "USB debugging" are enabled on your Android device
```

Comment:
I added the dot. Not sure what you mean by "end of the section"... anyway, checking  "Developer options" and "USB debugging" should be done before "adb install". Otherwise "adb install" will complain "Device not found".